osmosis-1/osmosis-1.assetlist.json

```diff
@@ -443,6 +443,35 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/ixo.png"
       }
+    },
+    {
+      "description": "Sifchain ROWAN Token",
+      "denom_units": [
+        {
+          "denom": "ibc/8B5FDA7B727F8E0C4B6E73A1DF529CE4C63208FD66FC207238A9ECB872C70419",
+          "exponent": 0,
+          "aliases": [
+            "xrowan"
+          ]
+        },
+        {
+          "denom": "rowan",
+          "exponent": 10,
```

Comment:
Guys, I'm pretty sure this is wrong.  Doesn't rowan use 18 decimal points of precission.